cpu/esp32/Makefile.include

```diff
@@ -1,114 +1,51 @@
 # check some environment variables first
 ifndef ESP32_SDK_DIR
-    $(info ESP32_SDK_DIR should be defined as /path/to/esp-idf directory)
-    $(info ESP32_SDK_DIR is set by default to /opt/esp/esp-idf)
-    export ESP32_SDK_DIR=/opt/esp/esp-idf
+  $(info ESP32_SDK_DIR should be defined as /path/to/esp-idf directory)
+  $(info ESP32_SDK_DIR is set by default to /opt/esp/esp-idf)
+  export ESP32_SDK_DIR=/opt/esp/esp-idf
 endif
 
-# DEFAULT compile configuration
+ESP_SDK_DIR = $(ESP32_SDK_DIR)
 
-# FLASH_MODE=[ dout | dio | qout | qio ]
-# use flash mode dout by default to keep GPIO9 and GPIO10 free for use
-export FLASH_MODE ?= dout
-
-# enable GDBSTUP for debugging on exceptions
-ifeq ($(ENABLE_GDBSTUB), 1)
-    USEMODULE += esp_gdbstub
-endif
-
-# enable GDB for compilation with debug info
-ifeq ($(ENABLE_GDB), 1)
-    USEMODULE += esp_gdb
-endif
-
-# enable modules at command line for testing
-ifneq ($(USE_MODULES), )
-    USEMODULE += $(USE_MODULES)
-endif
+# ESP32 specific flashing optoins
+FLASH_CHIP  = esp32
+FLASH_MODE ?= dout
+FLASH_FREQ  = 40m   # DO NOT CHANGE
+FLASH_SIZE ?= 4
+BOOTLOADER_POS = 0x1000
 
 # SPECIAL module dependencies
 # cannot be done in Makefile.dep since Makefile.dep is included too late
 
-ifneq (,$(findstring core_thread_flags,$(USEMODULE)))
-    USEMODULE += pthread
-endif
-
 ifneq (,$(filter cpp,$(FEATURES_USED)))
-    USEMODULE += esp_cxx
+  USEMODULE += esp_cxx
 endif
 
-ifneq (,$(filter esp_gdbstub,$(USEMODULE)))
-    USEMODULE += esp_gdb
+ifneq (,$(filter esp_now esp_wifi esp_spi_ram,$(USEMODULE)))
+  # if any WiFi interface or SPI RAM is used, the ESP-IDF heap has to be used 
```

Comment:
What about adding "as well" at the end of the sentence?

Reply:
Ok, thanks.